omod/pom.xml

```diff
@@ -62,6 +85,10 @@
 
 		<!-- End OpenMRS core -->
 
+		<dependency>
+			<groupId>joda-time</groupId>
```

Comment:
If I'm not wrong it should inherit from the main pom.xml. The scope is defined there as provided